### PR TITLE
Add events API to issues

### PR DESCRIPTION
**Pull request by raphink**

### CODE REVIEW

**Comment by aendrew** [comment]
The first bit is taken care of in #161, so please remove that. Please also add documentation and unit tests. Thanks!

**Comment by raphink** [comment]
As far as I can tell, #161 is for contributors (people who have written
code merged in the project), but my commit adds support for collaborators
(people with commit/admin rights on the project), which is different.

**Comment by raphink** [comment]
As far as testing, there are no tests for issues yet, and `listCollaborators` requires a user with priviledges on the repository, which is not the case of the test user (thankfully), so it'll always return a 403.

**Comment by aendrew** [comment]
Closed in error; reopening.

**Comment by AurelioDeRosa** [comment]
Hi @raphink. Are you still interested in updating this PR?

**Comment by raphink** [comment]
@aurelioderosa sure. What changes are still required?

File: github.js [hunk 1]
```diff
@@ -338,6 +338,16 @@
         });
       };
 
+      // List all collaborators of a repository
+      // --------
+
+      this.listCollaborators = function(cb) {
+        _request('GET', repoPath + "/collaborators", null, function(err, collabs) {
```
Comment (AurelioDeRosa) [inline github.js]: This line can be changed in:

`_request('GET', repoPath + "/collaborators", null, cb);`

**Comment by AurelioDeRosa** [comment]
This PR needs tests and also to add documentation to the README file. Thank you for the interest.

**Comment by raphink** [comment]
@AurelioDeRosa Any comment on https://github.com/michael/github/pull/180#issuecomment-97697439 ? Has the situation changed regarding testing since?

**Comment by AurelioDeRosa** [comment]
We have tests for issues now. You can find them in the `test` folder.

**Comment by raphink** [comment]
OK I'll have a look.

**Comment by raphink** [comment]
@AurelioDeRosa I have rebased and added tests

**Comment by raphink** [comment]
@AurelioDeRosa @clayreimann any news from this?

**Comment by clayreimann** [comment]
It's not merged yet (obviously) and two of the three APIs you're adding have been added already, but I'll get issue events in today or tomorrow.

**Comment by raphink** [comment]
Is collaborators added?

**Comment by clayreimann** [comment]
I believe so. We have both contributors and collaborators

**Comment by clayreimann** [comment]
@raphink you can see the docs for [contributors](http://michael.github.io/github/docs/1.1.0/Repository.html#getContributors) and [collaborators](http://michael.github.io/github/docs/1.1.0/Repository.html#getCollaborators)

**Comment by raphink** [comment]
Thanks. I'll have to rebase my code.

**Comment by clayreimann** [comment]
@raphink This is released with version `2.2.0`

**Comment by raphink** [comment]
Thank you.

**Pull request closed without merging**